js/communication/geppettoJupyter/PythonControlledCapability.js

```diff
@@ -13,15 +13,26 @@ define(function (require) {
 
   module.exports = {
     createPythonControlledComponent (WrappedComponent) {
+      if (typeof WrappedComponent !== "function") {
```

Comment:
I get what is happening since I already worked with this module but a comment here to explain what is happening would not harm :)

Reply:
The comment is up there: ` // Fixes components defined as objects (e.g. Material-ui components)`